src/components/Journeys/Journey/Timer.test.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow, mount } from 'enzyme';
+import sinon from 'sinon';
+import Timer, { timerExpired } from './Timer';
+
+describe('Timer', () => {
+  it('renders Timer', () => {
+    const seconds = 5;
+    const callback = () => {};
+
+    const component = mount(<Timer seconds={seconds} onComplete={callback} />);
```

Comment:
is there a reason why you need mount here instead of shallow? or anywhere else in your code for that matter.